setup.cfg

```diff
@@ -21,7 +21,6 @@ python_files = test/test_*.py
 norecursedirs=test/helpers
 filterwarnings =
 	once::Warning
-strict = true
```

Comment:
Just a quick comment as to why we dropped strict?  Didn't see this until now (totally un-related to issues on Copr though)

Reply:
It raised a `PytestConfigWarning: Unknown config option: strict`, so I removed it with 4f6e18a1 without thinking much about it. Apologies, feel free to revert!